front/components/quick_start_guide.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+  FolderIcon,
+  GlobeAltIcon,
+  Icon,
+  Modal,
+  Page,
+  RocketIcon,
+} from "@dust-tt/sparkle";
+import { CloudArrowLeftRightIcon } from "@dust-tt/sparkle";
+import type { UserType, WorkspaceType } from "@dust-tt/types";
+import { useEffect, useRef } from "react";
+
+import { getBrowserClient } from "@app/lib/amplitude/browser";
+
+export function QuickStartGuide({
+  owner,
+  user,
+  show,
+  onClose,
+}: {
+  owner: WorkspaceType;
+  user: UserType;
+  show: boolean;
+  onClose: () => void;
+}) {
+  const showedStartTimeRef = useRef<number | null>(null);
+
+  useEffect(() => {
+    // Track view duration on amplitude
+    if (show) {
+      showedStartTimeRef.current = Date.now();
+    } else {
+      if (showedStartTimeRef.current) {
+        const duration = Date.now() - showedStartTimeRef.current;
+        showedStartTimeRef.current = null;
+
+        const amplitude = getBrowserClient();
+        amplitude.identify(`user-${user.id.toString()}`);
+        amplitude.quickGuideView({
+          workspaceId: owner.sId,
+          workspaceName: owner.name,
+          duration,
+        });
+      }
+    }
+  }, [owner.name, owner.sId, show, user.id]);
+
+  return (
+    <Modal
+      isOpen={show}
+      variant="full-screen"
+      hasChanged={false}
+      onClose={onClose}
+      title="Quick Guide"
+    >
+      <div className="mb-12">
+        <Page>
+          <Page.Header
+            icon={RocketIcon}
+            title={
+              <>
+                Get Started:
+                <br />
+                Quick Guide for new members
+              </>
+            }
+          />
+
+          <Page.Horizontal>
+            <Page.Vertical sizing="grow">
+              <Page.H variant="h4">
+                👋 <br />
+                Hello <span className="text-success-500">@mentions</span>
+              </Page.H>
+              <Page.P size="md">
+                In Dust, you have access to{" "}
+                <strong>multiple AI&nbsp;assistants</strong>.
+              </Page.P>
+              <Page.P size="md">
+                You can call an assistant by using{" "}
+                <span className="italic">“mentions”</span> (Typing{" "}
+                <strong>"@"</strong> and the name of the assistant). You can
+                even call several assistants at the same time and in the same
+                thread.
+              </Page.P>
+            </Page.Vertical>
+            <Page.Vertical sizing="grow">
+              <img src="/static/quick_start_guide_input_bar.png" />
+            </Page.Vertical>
+          </Page.Horizontal>
+          <Page.Horizontal>
+            <Page.Vertical sizing="grow">
```

Comment:
Would re-order: 

Why multiple assistants? 
- **Specialized assistants can help answer questions, really well.** 
- Assistants in Dust can be provided by Dust, by your company (Company assistants) or by your coworkers (Shared assistants).
- Specialised assistants generally combine a model with specific instructions and access to knowledge.
- Raw model assistants, without any particular instructions or access to your company knowledge are also available, like @gpt4. 

cc @Duncid